test/images/dnsutils/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine
```

Comment:
cc @tianon would it be possible to get 3.6 images for armhf and aarch64 as well?